cfgov/v1/__init__.py

```diff
@@ -35,12 +39,30 @@ def environment(**options):
         'choices_for_page_type': ref.choices_for_page_type,
         'is_blog': ref.is_blog,
         'get_page_state_url': share.get_page_state_url,
+        'external_links': external_links,
     })
     env.filters.update({
         'slugify': slugify,
     })
     return env
 
+EXTERNAL_LINK_PATTERN = '(https?:\/\/(?:www\.)?(?![^\?]*(cfpb|consumerfinance).gov)(?!(content\.)?localhost).*)'
+
+
+def external_links(value):
+    soup = BeautifulSoup(expand_db_html(value.source))
+
+    try:
+        for a in soup('a'):
+            p = re.compile(EXTERNAL_LINK_PATTERN)
```

Comment:
yea so the regex also covers cases if anyone added a link like `https://www.facebook.com/?redirect=http%3A//test.cfpb.gov/the-bureau` with the conditional that would be considered a valid internal. 
i think for now lets leave it in, and post V1 we can look into a more performant to handle all the cases?